modules/API/pages/built-in-endpoints.adoc

```diff
@@ -1625,7 +1664,7 @@ $ curl -X POST --data-binary @add_id6.json \
   "code": "REST-0003"
 }
 ----
```

Comment:
When you use tabs, a tab is in an open block indicated by `--`. When you end one it needs to be closed. This one doesn't seem to be closed.